src/python_routing_v02/compiler.sh

```diff
@@ -10,6 +10,12 @@ build_reservoir_kernel=true
 build_framework=true
 build_routing=true
 
+if [ -z "$F90" ]
+then
+    export F90="gfortran"
+    echo "using F90='gfortran'"
+fi
+
```

Comment:
@groutr or @hellkite500 
Will you comment on the palatability of this insertion to catch the case where no F90 environment variable is in place?

Thanks...

Reply:
This is really just a dev utility, I don't see too much harm imposing a default.  Worst case, gfortran doesn't exist and a make error occurs.